aries-site/src/pages/components/data.mdx

```diff
@@ -0,0 +1,162 @@
+import { Example } from '../../layouts';
+import {
+  DataExample,
+  DataFilterExample,
+  DataFiltersExample,
+  DataSearchExample,
+  DataSortExample,
+  DataSummaryExample,
+  DataTableColumnsExample,
+  DataTableGroupByExample,
+  DataViewExample,
+  ToolbarExample,
+} from '../../examples/components/';
+
+Data is the wrapper component and orchestrator for a set of subcomponents. Because this set of components work together to create the entirety of the experience, we like to refer to them as "Data and friends".
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/data/DataSearchExample.js"
+  docs="https://v2.grommet.io/data"
+  template
+  height="auto"
+  showResponsiveControls={false}
+>
+  <DataExample />
+</Example>
+
+## Subcomponents
+
+### DataFilter
+
+An individual filter for a property. Based on the type of data for that property, DataFilter determines a suitable "smart default" for what kind of input to present.
```

Comment:
wonder if we can mention that the "smart default" is picked based on the chart below.